tests/utils/result_mapper.py

```
@@ -0,0 +1,131 @@
+
+        actual_nomenclature = ""
+        actual_similar_nomenclatures = ""
+        if mapping_result['mappings']:
+            actual_nomenclature = mapping_result['mappings'][0]['nomenclature']
+
+        if mapping_result['similar_mappings']:
+            for mapping in mapping_result['similar_mappings']:
+                actual_similar_nomenclatures += f"{mapping['nomenclature']}\n"
+
+        if actual_nomenclature == "":
+            if mapping_result['similar_mappings']:
```

Ruff

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)